config/tokens.json

```diff
@@ -939,6 +962,24 @@
         "type": "boxShadow",
         "description": "inner shadow primary button pressed"
       }
+    },
+    "text": {
+      "input-default": {
+        "value": "{text-md}",
+        "type": "typography"
+      },
+      "input-selected": {
+        "value": "{text-md+}",
```

Comment:
I know this has been there before, but something like `text-md+` will not work as a name: `text-md` and `text-md+` will overwrite each other, because special characters like `+` are being ignored during the token generation (transforming this file to CSS variables). Can we use something else instead?

Reply:
Ahh good to know! Thanks! I changed it now

Reply:
Hmm... okay cool, but now your other changes are gone. 🤔

Reply:
Yeah I was wondering about that ... when I opened the plugin today it didn't ask me to keep not-saved tokens nor pull from github because I had already pushed them but since they weren't on main, it didn't recognize them. :/ 

Is there a way to delete the last commit, merge the first to, and then I can pull them and do the changes of the third commit again afterwards?

Reply:
And yes, we could have reverted the last commit as well and then merge and do the steps again. I just thought this way it would be a bit faster.